virttest/bootstrap.py

```diff
@@ -350,6 +351,10 @@ def _forced_or_detected(forced, detected):
                                       "u%s" % detected.release)
         arch = _forced_or_detected(options.vt_host_distro_arch,
                                    "Host_arch_%s" % detected.arch)
+        vendor = cpu.get_vendor()
```

Comment:
This is not supported with older Avocado. Until all supported LTS versions support this `get_*` classes we need to use the old methods. Alternative solution would be to use `hasattr('get_vendor')` and use the new approach vs. old handling for the old Avocado versions.

Reply:
sure, makes sense.